pom.xml

```diff
@@ -651,17 +663,25 @@
                             </configuration>
                         </execution>
                     </executions>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.apache.maven.surefire</groupId>
+                            <artifactId>surefire-junit47</artifactId>
+                            <version>${maven.surefire.version}</version>
+                        </dependency>
+                    </dependencies>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.felix</groupId>
                     <artifactId>maven-bundle-plugin</artifactId>
-                    <version>4.2.1</version>
+                    <version>5.1.1</version>
```

Comment:
❓ Is it safe to assume we're on the next version of BND as well then? Any major header generation changes you noticed while upgrading?

Reply:
I am not sure what changed based on their change log.

https://github.com/apache/felix-dev/blob/master/tools/maven-bundle-plugin/changelog.txt

Reply:
It looks like they tie the version number directly to the BND version. I did have issues (though I do not remember exactly now) with the old version with Java 11 that were fixed by upgrading to 5.1.1.

https://github.com/apache/felix-dev/blob/master/tools/maven-bundle-plugin/pom.xml#L174

Reply:
Alright I'll just keep an eye on the manifests after this gets merged 👌